src/protocolsupport/protocol/pipeline/version/v_pe/PEPacketDecoder.java

```diff
@@ -13,6 +12,7 @@
 import protocolsupport.protocol.packet.middleimpl.ServerBoundPacketData;
 import protocolsupport.protocol.packet.middleimpl.serverbound.handshake.v_pe.ClientLogin;
 import protocolsupport.protocol.packet.middleimpl.serverbound.handshake.v_pe.Ping;
+import protocolsupport.protocol.packet.middleimpl.serverbound.play.v_8_9r1_9r2_10_11_12r1_12r2.CustomPayload;
```

Comment:
this whole situation still unnerves me. the PE client should be able to 'handle' the new-style key names, but theres some weird legacy mangling going on with bungeecord, or PSPE, thats making me using the old style one.

Reply:
I will merge master in a moment. Maybe this was the problem we fixed a bit ago still spooking around in mcpeinventory.